packages/common/src/services/gridState.service.ts

```diff
@@ -258,9 +258,9 @@ export class GridStateService {
    */
   getCurrentPagination(): CurrentPagination | null {
     if (this._gridOptions.enablePagination) {
```

Comment:
looks like _gridOptions can be null sometimes. If you check null on line 260 with "?.", you can remove the "?." on line 261.